File: angular/src/app/person.ts
export class Person {
  id: number = 0;
  lastname: string = "";
  firstname: string = "";
  email: string = "";
  address: string = "";
  phoneNumber: string = "";
  dob: string = "";
  gender: string = "";
  fingerprintCode: string = "";
  photo: string = "";
  photo_of_the_person: string = "";
}

export interface PersonPayload {
  lastname: string;
  firstname: string;
  email: string;
  address: string;
  phoneNumber: string;
  dob: string;
  gender: string;
}

export interface AddPersonDTO {
  profile: PersonPayload;
  password: string;
}

File: Desktop/projectt/lib/model/person.dart
class Person {
  final int? id;

  final String? firstname;

  final String? lastname;

  final String? email;

  final String? address;

  final String? phoneNumber;

  final String? dob;

  final String? gender;

  Person(
      {this.id,
      this.address,
      this.dob,
      this.email,
      this.firstname,
      this.gender,
      this.lastname,
      this.phoneNumber});

  factory Person.fromJson(Map<String, dynamic> json) {
    return Person(
      id: json['id'],
      firstname: json['firstname'],
      lastname: json['lastname'],
      email: json['email'],
      address: json['address'],
      dob: json['dob'],
      gender: json['gender'],
      phoneNumber: json['phoneNumber'],
    );
  }
}
